doc/man7/provider-cipher.pod

```diff
@@ -248,6 +248,11 @@ block has been "used" already.
 Gets or sets the AEAD tag for the associated cipher ctx.
 See L<EVP_EncryptInit(3)/AEAD Interface>.
 
+=item B<OSSL_CIPHER_PARAM_AEAD_TAGLEN> (int)
+
+Gets the tag length to be used for an AEAD cipher for the associated cipher ctx.
```

Comment:
Does setting make sense?  Truncated GCM tags are permitted but frowned upon.

Reply:
Currently we use hardwired values in the tests to get the length. This could also be frowned upon.

Reply:
See #8024 for a place when it might be needed.. If you have an alternative suggestion that is good.